.env

```diff
@@ -12,7 +12,8 @@ SESSION_REDIS_HOST=localhost
 SESSION_REDIS_PORT=6379
 STORAGE_REDIS_DB_INDEX=1
 STORAGE_REDIS_HOST=localhost
-STORAGE_REDIS_PORT=6379S3_BUCKET=
```

Comment:
@jbirdjavi I happened to notice this missing newline when trying local development